framework/ui/internal/uiconfiguration.cpp

```diff
@@ -18,17 +18,17 @@ UiConfiguration::UiConfiguration()
     settings()->addItem(FONT_FAMILY_KEY, Val("FreeSans"));
     settings()->addItem(FONT_SIZE_KEY, Val(12));
 
-    settings()->valueChanged(THEME_TYPE_KEY).onReceive(nullptr, [this] (const Val& val) {
-        m_currentThemeTypeChannel.send(static_cast<ThemeType>(val.toInt()));
-    });
+    settings()->valueChanged(THEME_TYPE_KEY).onReceive(nullptr, [this](const Val& val) {
+                m_currentThemeTypeChannel.send(static_cast<ThemeType>(val.toInt()));
+            });
```

Comment:
Do you need these indents?

Reply:
They were made by an autoformatting utility. I’m configured so that auto-formatting is called to save the file. In our cases, the formatting utility is crucial. If it does something wrong, then we need to do PR with a change in the config for her.